### ISSUE TITLE
 [BUG] App crashing due to UserFeedBackView trying to be added multiple times

### ISSUE THREAD

**Issue by imablanco** [applivery/applivery-android-sdk#57]
**Account information**
Organization Owner email address: alvaro.bl91@gmail.com
App ID:

**Describe the bug**
Due to a race condition, sometimes when taking a screenshoot the SDK is notified multiple times as per registered MediaStore's ContentObserver design, which leads towards trying to show UserFeedbackView on every notification. As UserFeedback is stored as a static field, the FragmentManager throws an Exception when trying to add the same exact Fragment more than once.

**To Reproduce**
1. Go to Sample App
2. Enable screenshot feedback
3. Take a screenshot
4. Eventually the crash will appear

**Expected behavior**
The app should handle this situation and provides some type of mutual exclusion strategy when between screenshots processing.
Also, UserFeedbackView should no be stored a static field because it is a memory leak.

**Smartphone (please complete the following information):**
 - Device: Pixel 2XL
 - OS: Android 10

**Additional context**
Crash stacktrace
```
 Process: com.applivery.sample, PID: 4914
    java.lang.IllegalStateException: Fragment already added: UserFeedbackView{198bf #7 }
        at android.app.FragmentManagerImpl.addFragment(FragmentManager.java:1714)
        at android.app.BackStackRecord.executeOps(BackStackRecord.java:770)
        at android.app.FragmentManagerImpl.executeOps(FragmentManager.java:2410)
        at android.app.FragmentManagerImpl.executeOpsTogether(FragmentManager.java:2205)
        at android.app.FragmentManagerImpl.removeRedundantOperationsAndExecute(FragmentManager.java:2166)
        at android.app.FragmentManagerImpl.execPendingActions(FragmentManager.java:2067)
        at android.app.FragmentManagerImpl$1.run(FragmentManager.java:742)
        at android.os.Handler.handleCallback(Handler.java:883)
        at android.os.Handler.dispatchMessage(Handler.java:100)
        at android.os.Looper.loop(Looper.java:214)
        at android.app.ActivityThread.main(ActivityThread.java:7356)
        at java.lang.reflect.Method.invoke(Native Method)
        at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:492)
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:930)
```

**Comment by mrebollob** [comment]
This should be fixed here: 42271da6a7e1fac51b6f739e7f44ab62c532e4c0

**Comment by CesarTrigo** [comment]
Thank you @imablanco!
